server/routes/induction/update/additionalTrainingUpdateController.test.ts

```diff
@@ -173,7 +170,6 @@ describe('additionalTrainingUpdateController', () => {
 
       // Then
       expect(res.render).toHaveBeenCalledWith('pages/induction/additionalTraining/index', expectedView)
-      expect(req.session.additionalTrainingForm).toBeUndefined()
```

Comment:
Do we no longer expect the form to be undefined? I can't see that the behaviour/code in the implementation has changed in this respect? 🤔

Reply:
Ah, OK 👍